### PR TITLE
Story11task1

### PR DESCRIPTION
**Pull request by Sipho-Ndlovu**
Added CSS styling for the project...

### CODE REVIEW

File: src/Components/HomePage/AllBooks/AllBooks.css [hunk 1]
```diff
@@ -1,10 +1,14 @@
 .all-books {
     display: flex;
     flex-wrap: wrap;
-    gap: 40px;
+    gap: 100px;
     padding: 80px;
+    justify-content: center;
+}
+.nav {
```
Comment (Andrei-Trifu) [inline src/Components/HomePage/AllBooks/AllBooks.css:9]: maybe add a whitespace between classes so they're more visible

File: src/Components/HomePage/AllBooks/BookListing/BookListing.css [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
     justify-content: center;
     flex-direction: column;
     align-items: center;
-    box-shadow: 2px 2px 5px gray;
+    background-color: rgba(141, 141, 141, 0.5);
+    box-shadow: 2px 2px 5px black;
+    border-radius: 15px;
     width: 400px;
 }
-
```
Comment (Andrei-Trifu) [inline src/Components/HomePage/AllBooks/BookListing/BookListing.css:11]: whitespaces here as well

File: src/components/BookDetail/BookDetail.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    display:grid;
+    grid-template-columns: 1fr 3fr;
+    color: white;
+    background-color: rgba(141, 141, 141, 0.5);
+    border-radius: 15px;
+    padding: 20px;
+    margin: 30px;
+    box-shadow: 2px 2px 5px black;
+}
+.detail-container-text{
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.css:12]: whitespace

File: src/components/BookDetail/BookDetail.jsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                        <p>{bookAuthor}</p>
+                        <p>{bookYear}</p>
+                        <p>{bookPageCount}</p>
+                        <p>{bookGenre}</p>
+                        <p>{bookBlurb}</p>
+                    </div>
+                </div>                
+            } 
+
+
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.jsx]: remove some of these whitespaces

File: src/components/BookDetail/BookDetail.jsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        </div>
+
+    )
+
```
Comment (Andrei-Trifu) [inline src/components/BookDetail/BookDetail.jsx]: remove some of these whitespaces

**Review by Andrei-Trifu — APPROVED** [review]
Looks good! Maybe just some whitespaces to remove.

File: src/Components/HomePage/AllBooks/AllBooks.css (hunk 1 above)
Comment (Sipho-Ndlovu) [reply src/Components/HomePage/AllBooks/AllBooks.css:9]: Done, whitespaces added.

File: src/Components/HomePage/AllBooks/BookListing/BookListing.css (hunk 2 above)
Comment (Sipho-Ndlovu) [reply src/Components/HomePage/AllBooks/BookListing/BookListing.css:11]: Done, whitespaces added.

File: src/components/BookDetail/BookDetail.css (hunk 3 above)
Comment (Sipho-Ndlovu) [reply src/components/BookDetail/BookDetail.css:12]: Done, whitespaces added.

File: src/components/BookDetail/BookDetail.jsx (hunk 4 above)
Comment (Sipho-Ndlovu) [reply src/components/BookDetail/BookDetail.jsx]: Done, whitespaces removed.

File: src/components/BookDetail/BookDetail.jsx (hunk 5 above)
Comment (Sipho-Ndlovu) [reply src/components/BookDetail/BookDetail.jsx]: Done, whitespaces removed.

**Pull request merged**